catalog/app/containers/Auth/onelogin-logo.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@dimaryaz pls optimize this svg with `svgo` (`npx svgo --multipass  -i app/containers/Auth/onelogin-logo.svg`)

Reply:
Ok